packages/ag-charts-enterprise/src/features/zoom/zoomToolbar.ts

```diff
@@ -78,6 +89,30 @@ export class ZoomToolbar {
         } else if (typeof time === 'function') {
             rangeX.updateWith(time);
         }
+
+        const range = rangeX.getRange();
+        this.selectedZoom = range != null ? { id: event.id, min: range.min, max: range.max } : undefined;
+    }
+
+    private onZoomChanged(e: _ModuleSupport.ZoomChangeEvent) {
+        // SetTimeout required because we set selected zoom after the event is called
+        setTimeout(() => {
+            const { selectedZoom } = this;
+            const { x } = e;
+
+            this.toolbarManager.toggleGroup('zoom-toolbar', 'ranges', { active: false });
+
+            if (
+                selectedZoom != null &&
+                x != null &&
+                Math.abs(selectedZoom.min - x.min) < 1e-9 &&
+                Math.abs(selectedZoom.max - x.max) < 1e-9
```

Comment:
nit: Could we change this to `1e-10` to match `isZoomEqual()` in `zoomUtils.ts`?

I've had bugs with zooms not matching and refining the granularity of the floating point of zoom to be the same everywhere.